analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.IO;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using ITs.JsonParser.Json;
+using ITs.JsonParser.Reports;
+
+namespace ITs.JsonParser;
+
+/// <summary>
+/// Reads SARIF reports from "output/" and writes custom reports to "actual/".
+/// </summary>
+public static class IssueParser
+{
+    private static readonly Regex RxSonarRule = new("^S[0-9]+$", RegexOptions.Compiled);
```

Comment:
There's nothing to gain on regex like this, for the pretty huge price it costs.
```suggestion
    private static readonly Regex RxSonarRule = new("^S[0-9]+$");
```